buildspecs/build.yml

```diff
@@ -0,0 +1,67 @@
+version: 0.2
+
+env:
+  variables:
+    # Github username of the forked repo on which to make builds
+    GITHUBUSERNAME: aws
+ 
+phases:
+  install:
+    commands:
+       - architecture=""
+       # Same buildspec for different architectures - detect the architecture here and rename the artifacts accordingly
+       - case $(uname -m) in
+           x86_64)
+             architecture="amd64"
+           ;;
+           aarch64)
+             architecture="arm64"
+           ;;
+         esac
+
+       # Tested with GOVERSION 1.15.x and 1.17.x
+       - GOVERSION="1.15.15"
```

Comment:
is go 1.17 available -- can we default to that?

Reply:
Yep, will update to 1.17

Reply:
Reading from GO_VERSION in the repo now, pushed changes to address this